client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -79,7 +79,18 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.EMAIL_VERIFIED:
 			taskData = {
 				timing: 1,
-				title: translate( 'Confirm your email address' ),
+				title: translate(
+					'Confirm your email address {{emailWrapper}} %(email)s {{/emailWrapper}} ',
```

Comment:
If we put the `email` inside the `emailWrapper` component, we could have a stronger string, something like `'Confirm your email address {{emailAddress}}`. It would also help make translating the string a bit easier and less error prone.

Reply:
Good point, also revising https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md to really understand this.